classes/core-image-pelux.bbclass

```diff
@@ -1,5 +1,5 @@
 #
-#   Copyright (C) 2017 Pelagicore AB
+#   Copyright (C) 2018 Luxoft Sweden AB
```

Comment:
You cannot change copyright!

Reply:
When doing this a few weeks ago the consensus is that if the file has "Pelagicore" then we add a line with "Luxoft Sweden". If there is a line with a previous year and "Luxoft Sweden" then leave as is, since the Copyright is valid for many many years.

Reply:
Correct. But this is not what has been done.

Reply:
Yes, that was also my point.

Reply:
So in this case I should revert back to "Copyright (C) 2017 Pelagicore AB"? Its not clear to me from the above comments what to do.

Reply:
Sorry about the confusion, no I mean that you should add a line with "Luxoft Sweden" and 2018.